src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -63,8 +63,7 @@ public static async Task<RemoteHostClient> CreateAsync(Workspace workspace, bool
             _inprocServices = inprocServices;
             _remotableDataRpc = remotableDataRpc;
 
-            _rpc = new JsonRpc(new JsonRpcMessageHandler(stream, stream), target: this);
-            _rpc.JsonSerializer.Converters.Add(AggregateJsonConverter.Instance);
```

Comment:
>AggregateJsonConverter.Instance [](start = 47, length = 31)

You can remove AggregateJsonConverter.Instance from AggregateJsonConverter

Reply:
I see you still use it in Extensions. Maybe you should move it there.

---
In reply to: [277095163](https://github.com/dotnet/roslyn/pull/34860#discussion_r277095163) [](ancestors = 277095163)

Reply:
@ivanbasov I am not sure what you meant by this?

Reply:
@heejaechang you removed some lines 
`_rpc.JsonSerializer.Converters.Add(AggregateJsonConverter.Instance);`
in different files.

All them refer to `AggregateJsonConverter.Instance`. As far as I understand, now `AggregateJsonConverter.Instance` is used just once. It is used by `Extensions` and can be moved there.

Reply:
this one is shared by multiple projects as linked file. so moving whole code to Extension.cs make this file to big. so, I would prefer leaving things as it is for now.